src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageDetailsTabViewModel.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Shell;
+using NuGet.VisualStudio;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.PackageManagement.UI.ViewModels
+{
+    public class PackageDetailsTabViewModel : ViewModelBase, IDisposable
+    {
+        private bool _disposed = false;
+        private bool _readmeTabEnabled;
+
+        public ReadmePreviewViewModel ReadmePreviewViewModel { get; private set; }
+
+        public DetailControlModel DetailControlModel { get; private set; }
+
+        public ObservableCollection<TitledPageViewModelBase> Tabs { get; private set; }
+
+        private TitledPageViewModelBase _selectedTab;
+        public TitledPageViewModelBase SelectedTab
+        {
+            get => _selectedTab;
+            set
+            {
+                SetAndRaisePropertyChanged(ref _selectedTab, value);
+            }
```

Comment:
Nit: could be more concise.

```suggestion
            set => SetAndRaisePropertyChanged(ref _selectedTab, value);
```